test/Microsoft.Azure.WebJobs.Host.EndToEndTests/AsyncChainEndToEndTests.cs

```diff
@@ -147,13 +152,24 @@ public async Task AsyncChainEndToEnd()
 
                 // Validate Logger output
                 var allLogs = _loggerProvider.CreatedLoggers.SelectMany(l => l.GetLogMessages().SelectMany(m => m.FormattedMessage.Trim().Split(new string[] { Environment.NewLine }, StringSplitOptions.None))).OrderBy(p => p).ToArray();
-                // Logger doesn't log the 'Executing' messages
-                var loggerExpected = expectedOutputLines.Where(l => !l.StartsWith("Executing '")).ToArray();
+                // Logger doesn't log the 'Executing' or 'Trigger Details' messages
```

Comment:
I'd forgotten about this -- it kinda stinks. This means that people using App Insights (which uses ILogger) won't see these logs in functions v1/webjobs v2. I can't remember why this is the case -- can you file an issue so that we can follow up? I'd rather not hold this up as this will still give us valuable details in our backend logs for investigations.

Reply:
Sounds good. I'll create a separate issue to track this.